### DISCUSSION TITLE
[2024-04-10] This Bi-Weekly in Dora-rs

### DISCUSSION THREAD

**Discussion by haixuanTao** [dora-rs/dora#469] [Status Update]
# Future bi-weekly are going to be on our discord at: https://discord.gg/Ypjnc9r47n

## Demo

Post from Thomas Wolf, Cofounder of HuggingFace: 
- https://www.linkedin.com/feed/update/urn:li:activity:7182825998869000192?updateEntityUrn=urn%3Ali%3Afs_feedUpdate%3A(V2%2Curn%3Ali%3Aactivity%3A7182825998869000192)
- https://twitter.com/Thom_Wolf/status/1777025381020688470

Future demo coming up!

**Comment by haixuanTao** [comment]
## Addition

- refactor `README` to keep up with our latest development
- Adding  #468 to enable specific conda environment when running dora operators
- releasing a v0.3.3 version with our latest changes
- fixed cmake example #460 thanks to @XxChang

**Comment by haixuanTao** [comment]
## Community

- Put down rerun.io as a hoped for visualization
- Put down couple of hardwares in the list of hoped for,  especially robotic arms as https://tonyzhaozh.github.io/aloha/ showcase manipulation using low cost arms (~20k $ ). If dora-rs can provide a (~1k $) reproduction of `aloha` setup with wheels, it could be a very powerful setup for dora-rs.

**Comment by binary-bruce** [comment]
is it possible to receive this Bi-Weekly by email subscription?

**Reply by phil-opp** [reply]
You could use GitHub's "watch" feature with a custom config to only watch "discussions". Then GitHub will notify you for all new discussion threads. Unfortunately, it's not possible to only subscribe to a specific discussion category, as far as I know.

**Comment by put-down-crate** [comment]
There are a few differences between   "Guides/Getting Started/Python Conversation/1. Create a new dataflow"
and  "command `dora new conversation_py` result"

I was  dora-cli 0.3.5


guide command result is 
.
├── dataflow.yml
├── node_1
│   └── node_1.py
├── op_1
│   └── op_1.py
└── op_2
    └── op_2.py


dora-cli 0.3.5 command result is 
└─conversation_py
    ├─listener_1
    ├─talker_1
    └─talker_2
